modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -131,5 +133,15 @@ def expand_va_url(url)
 
       "#{FORM_BASE_URL}/vaforms/#{url.gsub('./', '')}" if url.starts_with?('./va') || url.starts_with?('./medical')
     end
+
+    def notify_slack(old_form_url, new_form_url, form_name)
+      return unless Settings.va_forms.slack.enabled
+
+      @slack_url = Settings.va_forms.slack.notification_url
+      @slack_users = Settings.va_forms.slack.users
+      Faraday.post(@slack_url,
```

Comment:
Faraday post can drop an exception under certain circumstances, like failing to get a connection.  It appears you have important work to continue after the call?  Perhaps wrap in a begin / rescue trap Faraday::ClientError and log it ( Rails.logger.error("Whoopsy", e) )